emission/tests/modellingTests/modellingTestAssets.py

```diff
@@ -189,22 +200,23 @@ def generate_mock_trips(
             purpose_weights=label_data.get('purpose_weights')
         )
         trip = build_mock_trip(user_id, o, d, labels, start_ts, end_ts)
-        trip = add_trip_demographics(trip)
+        if survey_data is not None:
+            trip = add_trip_demographics(trip, survey_data)
         result.append(trip)
         
     random.shuffle(result) 
     return result
 
 
-def add_trip_demographics(trip):
-    trip['data']['survey'] = {}
-    survey_features = {
-        'hhinc':['0-24999','25000-49000','50000-99999','100000+'],
-        'age':[x for x in range(0, 70)],
-        'veh':['0','1','2','3','4+']
-    }
+def add_trip_demographics(trip, survey_features):
+    response_id = ''.join(random.choices(string.ascii_uppercase + string.ascii_lowercase, k=22))
+    trip['data']['jsonDocResponse'] = {response_id: {'group_yk8eb99': {}, 'group_hg4zz25': {}, 'group_pa5ah98': {}}}
```

Comment:
again, not sure why the demographic responses are associated with trips? You don't seem to be `copy()`ing the data, so presumably we are only storing links to the same demographic information, not actually creating duplicates that blow up the memory usage